app/models/event.rb

```diff
@@ -27,10 +28,14 @@ def location
   end
 
   def title_with_date
-    date = starts_at.strftime("%B %d")
+    return nil if starts_at.blank?
```

Comment:
@itsWill slightly more idiomatic Ruby to just do a return -- `nil` would be implied there. Example:

```
return if starts_at.blank?
```